addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -347,6 +347,18 @@ export class KanbanRecord extends Component {
             case "open": {
                 return openRecord(record);
             }
+            case "archive": {
+                this.dialog.add(ConfirmationDialog, {
+                    body: _t("Are you sure that you want to archive this record?"),
+                    confirmLabel: _t("Archive"),
+                    confirm: () => record.archive(),
+                    cancel: () => {},
+                });
+                return true
+            }
+            case "unarchive": {
+                return record.unarchive();
+            }
```

Comment:
@aab-odoo Hi, wouldn't it make more sense to get the archive and unarchive from the props here like we do for openRecord and deleteRecord to allow easier behavior customization ?

Reply:
@jugj-odoo @mepe-odoo Indeed, we could do that :+1: